.github/workflows/ci-cd.yml

```diff
@@ -215,6 +215,9 @@ jobs:
     secrets:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      VITE_E2E_WALLET_KEY: ${{ secrets.VITE_E2E_WALLET_KEY }}
+      VITE_E2E_WALLET_MNEMONIC: ${{ secrets.VITE_E2E_WALLET_MNEMONIC }}
+      VITE_E2E_WALLET_ADDRESS: ${{ secrets.VITE_E2E_WALLET_ADDRESS }}
```

Comment:
@drklee3 - do you know if I need this duplication or if putting the e2e secrets at this level will suffice?

https://github.com/Kava-Labs/oros/blob/05a8b5416d1cb66b21c07378b28e5abd3560c0e9/.github/workflows/ci-cd.yml#L247-L250